tests/integration/test_scheduler.py

```diff
@@ -237,3 +245,108 @@ async def test_no_poll_waiting_tasks(
 
     # For good measure, check the faked running task is reported at shutdown.
     assert "Orphaned task jobs:\n* 1/one (running)" in log.messages
+
+
+@pytest.mark.parametrize('reload', [False, True])
+@pytest.mark.parametrize(
+    'test_conf, expected_msg',
+    [
+        pytest.param(
+            {'Alan Wake': "It's not a lake, it's an ocean"},
+            "IllegalItemError: Alan Wake",
+            id="illegal item"
+        ),
+        pytest.param(
+            {
+                'scheduling': {
+                    'initial cycle point': "2k22",
+                    'graph': {'R1': "a => b"}
+                }
+            },
+            ("IllegalValueError: (type=cycle point) "
+             "[scheduling]initial cycle point = 2k22 - (Invalid cycle point)"),
+            id="illegal cycle point"
+        )
+    ]
+)
+async def test_illegal_config_load(
+    test_conf: dict,
+    expected_msg: str,
+    reload: bool,
+    flow: Callable,
+    one_conf: dict,
+    start: Callable,
+    run: Callable,
+    scheduler: Callable,
+    log_filter: Callable
+):
+    """Test that ParsecErrors (illegal config) - that occur during config load
+    when running a workflow - are displayed without traceback.
+
+    Params:
+        test_conf: Dict to update one_conf with.
+        expected_msg: Expected log message at error level.
+        reload: If False, test a workflow start with invalid config.
+            If True, test a workflow start with valid config followed by
+            reload with invalid config.
+    """
+    if not reload:
+        one_conf.update(test_conf)
+    reg: str = flow(one_conf)
+    schd: Scheduler = scheduler(reg)
+    log: pytest.LogCaptureFixture
+
+    if reload:
+        one_conf.update(test_conf)
+        run_dir = Path(get_workflow_run_dir(reg))
+        async with run(schd) as log:
```

Comment:
If we can avoid queueing tasks we don't need to run the main loop:

```suggestion
        async with start(schd) as log:
```